### DISCUSSION TITLE
Any EQ options available per speaker?

### DISCUSSION THREAD

**Discussion by griffindodd** [snapcast/snapcast#1230] [Q&A]
I LOVE THIS PROJECT!!!

I have 12 speakers on Snapcast now and just love this system. I do wonder though if there are any ways to do EQ tuning at the individual speaker level as some of my speakers do not have the option at the end point.

**Comment by DerPicknicker** [comment]
Camilladsp .. @badaix .. does snapcast support loopback devices? I have some problems to sync it with my other clients. Buffer is 1000ms

**Reply by badaix** [reply]
https://github.com/badaix/snapcast/blob/develop/doc/configuration.md#available-parameters-3

**Reply by DerPicknicker** [reply]
@badaix I know. The client uses an loopback as source but unfortunately with the DSP it's hard to sync. I never get the sync as without loopback + camilladsp. 

Do you have any suggestions which things I can optimise? Bigger buffer 2500ms+?
